python/lsst/ts/externalscripts/maintel/warmup_hexapod.py

```diff
@@ -0,0 +1,278 @@
+            $schema: http://json-schema/draft-07/schema#
+            $id: https://github.com/lsst-ts/ts_standardscripts/maintel/WarmUpHexapod.yaml
+            title: WarmUpHexapod v1
+            description: Configuration for WarmUpHexapod.
+            type: object
+            properties:
+              hexapod:
+                description: >-
+                  Which hexapod will move? "camera" for the camera hexapod
+                  or "m2" for the m2 hexapod.
+                type: string
+                enum: ["camera", "m2"]
+              axis:
+                description: Which axis will move? (x, y, z, u, v, w).
+                type: string
+                enum: ["x", "y", "z", "u", "v", "w"]
+                default: z
+              step_size:
+                description: >-
+                  The discrete step size in microns in which the hexapod will move.
+                  This can be a number or an array of numbers.
+                anyOf:
+                  - type: number
+                  - type: array
+                default: 250
+              sleep_time:
+                description: >-
+                  The sleep time in seconds between movements. The number of
+                  elements must match the number of elements of step_size.
+                anyOf:
+                  - type: number
+                    exclusiveMinimum: 0.0
+                  - type: array
+                    items:
+                      type: number
+                      exclusiveMinimum: 0.0
+                default: 1
+              max_position:
+                description: Maximum position (absolute value) for the movements.
+                type: number
+                exclusiveMinimum: 0.0
+                maximum: 13000.0
+                default: 13000.0
+            additionalProperties: false
+            """
+        return yaml.safe_load(yaml_schema)
+
+    async def configure(self, config):
+        """Configure the script.
+
+        Parameters
+        ----------
+        config: `types.SimpleNamespace`
+            Configuration data. See `get_schema` for information about data
+            structure.
+        """
+        self.log.debug(
+            f"Setting up configuration: \n"
+            f"  hexapod: {config.hexapod}\n"
+            f"  axis: {config.axis}\n"
+            f"  step_size: {config.step_size}\n"
+            f"  sleep_time: {config.sleep_time}\n"
+            f"  max_position: {config.max_position}\n"
+        )
+
+        self.config = config
+
+        # Converts the string to salindex
+        self.hexapod_name = f"{config.hexapod}_hexapod"
+        self.hexapod_sal_index = getattr(SalIndex, self.hexapod_name.upper())
+        self.hexapod = getattr(self.mtcs.rem, f"mthexapod_{self.hexapod_sal_index}")
+        self.log.debug(
+            f"Using the {self.hexapod_name} with sal index {self.hexapod_sal_index}"
+        )
+
+        # Assert step_size and sleep_time are compatible
```

Comment:
"Assert" usually means we might expect an `AssertionError`, which is not really happening here. 

Maybe reword to like:

> Make sure step_size and sleep_time are arrays.